### DISCUSSION TITLE
Nonexistent createClient export

### DISCUSSION THREAD

**Discussion by stretch07** [supabase/supabase#5492] [Questions] [answered]
![image](https://user-images.githubusercontent.com/78528552/154340670-f9545923-8aa4-423b-9706-aefeba916f56.png)
Not sure how to fix this, suggestions?

**Comment by stretch07** [comment]
Bumping this; does anyone have a solution?

**Comment by inian** [comment, marked as answer]
[This](https://github.com/supabase/supabase-js#umd) should work @its-pablo 
```html
<script src="https://unpkg.com/@supabase/supabase-js"></script>
```

```js
const { createClient } = supabase
```
